.gitignore

```diff
@@ -3,4 +3,6 @@ node_modules
 .next
 out
 .env
-*.sqlite
\ No newline at end of file
+*.sqlite
+hashicorp.sqlite-shm
+hashicorp.sqlite-wal
```

Comment:
We should really just include these in our template... 🤦🏻

Reply:
If it's okay, I can add a PR to the template for this and to remove the comment to get the Dato API token in discord:

<img width="836" alt="Screenshot 2025-05-21 at 5 36 33 PM" src="https://github.com/user-attachments/assets/710a5d66-96fc-4b95-81b6-61eb1aacaf20" />

Reply:
We already removed that comment and there is no reason for you to do work for us, but thank you.